### ISSUE TITLE
rumble stops working seemingly perminantly if scripting window is opened

### ISSUE THREAD

**Issue by steventylerseanherr** [mgba-emu/mgba#3296]
in current dev (mGBA-build-2024-09-19-win64-8523-49fa1a30c581cd868365c76d6136885a1bdcf238)
if you open the scripting window then all rumble input (output?) stops seemingly permanently

steps to reproduce:
open mgba
run this rom https://github.com/evanbowman/gba-rumble/releases/tag/1.0
either with native rumble or gbp
confirm rumble is working and then open the scripting window

**Comment by steventylerseanherr** [comment]
just tested 0.10.4 and this might be fixed?
ill let others confirm if thats the case

**Comment by endrift** [comment]
No, 0.10.4 is an older codebase than 0.11, with just some fixes cherry-picked. I haven't looked into fixing it on 0.11 yet.
